pye3d/detector_3d.py

```diff
@@ -148,6 +148,25 @@ def long_term_mode(self, mode: DetectorMode):
         if needs_reset:
             self.reset()
 
+    @property
+    def is_long_term_model_frozen(self) -> bool:
```

Comment:
Maybe you rather want to call this 

long_term_model_is_frozen

? 

Otherwise it sounds like a question and not like a property name to me.

Reply:
I actually trying to keep the naming consistent in the regard that I prefix all boolean properties with `is_` to make it immediately clear that it is supposed to be a boolean. I agree that your proposal would read more like actual english in case of an if-statement.
```py
if self.long_term_model_is_frozen:
    pass
```
I feel like the recognition of it being a boolean is worth the grammar-readability sacrifice.

Reply:
So actually, your version sounds more like a question than mine. But in any case, calling this as a question makes sense to me, as the response is either True or False. Setting the property does not as nice though. Your version would read nicer:
```py
self.is_long_term_model_frozen = True
# vs
self.long_term_model_is_frozen = True
```
My main argument remains the consistency with other boolean variables/properties that share the `is_` prefix.

Reply:
I totally agree, consistency trumps other considerations.